src/main/java/seedu/address/logic/commands/ListAllCommand.java

```diff
@@ -0,0 +1,23 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+
+/**
+ * Lists all items in Salesy.
+ */
+public class ListAllCommand extends Command {
+    public static final String COMMAND_WORD = "listAll";
+
+    public static final String MESSAGE_SUCCESS = "Listed all items in Salesy!";
```

Comment:
Should the message use the word "items" when it may be confused for SupplyItem although the command is listing all entities? Would "data" or "entities" be better?

Reply:
Yeah we should

Reply:
```suggestion
    public static final String MESSAGE_SUCCESS = "Listed all entities in Salesy!";
```